src/Spain/Mews.Fiscalizations.Spain.Tests/Basics.cs

```diff
@@ -82,8 +82,8 @@ private async Task AssertNifLookup(INonEmptyEnumerable<NifInfoEntry> entries, Ni
             var validator = new NifValidator(Certificate, httpTimeout: TimeSpan.FromSeconds(30));
             var response = await validator.CheckNif(new Request(entries));
 
-            Assert.AreEqual(response.SuccessResult.Results.Count(), entries.Count());
-            foreach (var result in response.SuccessResult.Results)
+            Assert.AreEqual(response.SuccessResult.Get().Results.Count(), entries.Count());
+            foreach (var result in response.SuccessResult.Get().Results)
```

Comment:
These 4 lines would also simplify to just `response.Get()` rather than `response.SuccessResult.Get()`